_drafts/SC1-4-1-link-style-in-text.md

```diff
@@ -20,14 +20,14 @@ This rule checks that links that are embedded in a block of text can be distingu
 - [G183: Using a contrast ratio of 3:1 with surrounding text and providing additional visual cues on focus for links or controls where color alone is used to identify them](http://www.w3.org/TR/2013/NOTE-WCAG20-TECHS-20130905/G183)
 - [C15: Using CSS to change the presentation of a user interface component when it receives focus](http://www.w3.org/TR/2014/NOTE-WCAG20-TECHS-20140916/C15:)
 - [Color contrast (WCAG 2.0 definition)](http://www.w3.org/TR/WCAG20/#contrast-ratiodef)
+- [Axe-core implementation of link-in-text-block](https://dequeuniversity.com/rules/worldspace/2.0/link-in-text-block)
 
 ## Assumptions
 
-- This test assumes that the 3:1 contrast difference between text is minimal to what would be sufficient to meet WCAG 2.0. This value is part of G183 technique, but is not specified in the 1.4.1 success criterion.
-- This tests assumes that one of the following shorthand CSS-properties (or the related expanded properties) is used to make the link visually evident: `background`, `border`, `color`, `font`, or `text-decoration`.
-- This test assumes that any use of border will make links sufficiently distinguishable
-- This test assumes that the different font is presented to the user.
-- Use of a `border`, of 1 or more pixels, not set to none, and not with a color of transparent, is assumed to be distinguishable
+- This test assumes that the 3:1 contrast difference between text is minimal to what would be sufficient to meet WCAG 2.0. This value is part of technique G183, but is not specified in the 1.4.1 success criterion.
+- This tests assumes that one of the following shorthand CSS properties (or the related expanded properties e.g. `background-color`) is used to make the link visually evident: `background`, `color`, `font`, or `text-decoration`.
```

Comment:
May be want to say >> This test assumes that the 3:1 contrast difference between the link text and the surrounding text << to make more clear.